### PR TITLE
Add publish workflow

### PR DESCRIPTION
**Pull request by aht007**
This PR adds the publish workflow to release the package to PyPI. The PR also changes the Package name because old name is already taken on PyPI.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
     long_description = fh.read()
 
 setuptools.setup(
-    name="olxcleaner",
+    name="openedx-olxcleaner",
```
Comment (aht007) [inline setup.py:13]: Changing the distribution name because `olxcleaner` name is already taken on PyPI

Comment (jolyonb) [reply setup.py:13]: I'm happy to take down that package - better for the actively maintained version to be there.

Comment (aht007) [reply setup.py:13]: Yeah sure, if that's the case I would go ahead and revert the package name in this PR. Thanks the rapid response.

Comment (jolyonb) [reply setup.py:13]: Actually, can I just add the open edx account as a collaborator? What would the username be?

Comment (jolyonb) [reply setup.py:13]: I found the "edx" user and sent an invite to be a maintainer on the package on pypi. The invite is pending acceptance.

Comment (aht007) [reply setup.py:13]: @jmbowman If you could help us out here...

Comment (jmbowman) [reply setup.py:13]: Sorry for the late reply.  The password I have on file for the edx PyPI user is outdated, you'll probably need an SRE ticket for this.

Comment (jolyonb) [reply setup.py:13]: @aht007 @jmbowman Let me know if I can be helpful here...

Comment (feanil) [reply setup.py:13]: @jolyonb maybe the invitation expired, can you send it again to the `edx` user on PyPI?

Comment (jolyonb) [reply setup.py:13]: @feanil Resent.

Comment (feanil) [reply setup.py:13]: @jolyonb I still don't see it.  Can you try one more time?  or try to add the `openedx` user instead?

Comment (jolyonb) [reply setup.py:13]: @feanil Just reinvited edx and openedx

Comment (feanil) [reply setup.py:13]: Got it! I've accepted the invite as `openedx`.  I'll now remove you as the owner.  Thank you for all the work so far!

**Comment by aht007** [comment]
@openedx/edx-infinity Can someone from the team please merge this PR and trigger a new Github release(that will also release the package to PyPI) as I don't have access here. Thanks

**Review by kdmccormick — APPROVED** [review]

**Pull request merged**